README.md

```diff
@@ -1306,30 +1306,30 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   ```swift
   // WRONG
-  public extension Foo {
```

Comment:
I like `Foo` and `Bar` and don't intend to eliminate them from my "vocabulary" (e.g. see [this post](https://stackoverflow.com/questions/64072242/why-cant-i-pass-an-object-conforming-to-a-protocol-to-a-function-that-expects-a) I made today) but I think more human examples will make this guide accessible to a broader audience 👍